databases/week3/meal_reservation_database.sql

```diff
@@ -0,0 +1,143 @@
+
+/*insert data into meal table*/
+insert into meal(`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+ values('chicken vindaloo','indian spicy chicken curry','valby','2021-08-08 04:49:05','1','120.23','2017-08-08 19:40:23'),
+ ('paneer curry','indian spicy paneer curry','søborg','2021-06-08 04:49:05','12','112.13','2019-08-05 15:40:23'),
+ ('palak curry','palak spicy paneer curry','virum','2015-06-08 04:49:05','10','110.28','2016-08-06 14:40:23');
+
+/*insert data into reservation table*/
+INSERT INTO reservation(`number_of_guests`,`meal_id`,`contact_phonenumber`,`contact_name`,`contact_email`)
+VALUES('5','1','45897890','john','john@gmail.com'),('6','1','75897890','mads','mads@gmail.com'),('7','3','35897890','Lone','Lone@gmail.com'),
+('8','2','95897890','sham','sham@gmail.com');
+
+/*insert data into review table*/
+INSERT INTO review(`title`,`description`,`meal_id`,`stars`)
+values('tasty','tasty spicy chicken curry','1','4'),('quantity very good','delicious spicy paneer curry','2','5'),('nice experience','delicious spicy palak curry','2','5');
+
+/*Get all meals*/
+SELECT * FROM meal;
+
+/* add new meal*/
+INSERT INTO meal(`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`)
+ values('daal tadka', 'yellow indian daal','lyngby','2021-08-08 04:49:05','11','119.23','2020-08-08 20:40:23');
+
+--Meal queries
+ /*Get a meal with any id, fx 1*/
+ SELECT * FROM meal WHERE id = 2;
+
+ /*Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes*/
+UPDATE meal
+SET title = 'butter chicken', description = 'indian spicy buttter chicken masala' 
+WHERE id = 5;
+
+/*Delete a meal with any id, fx 1*/
+DELETE FROM meal WHERE id = 5;
+
+--reservation queries
+
+/*Add a new reservation*/
+INSERT INTO reservation(`number_of_guests`,`meal_id`,`contact_phonenumber`,`contact_name`,`contact_email`)
+VALUES('5','1','45897890','mark','mark@gmail.com'),('8','1','45897890','Kone','kone@gmail.com');
+/*Get a reservation with any id, fx 1*/
+SELECT * FROM reservation WHERE id=1;
+
+/*Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes*/
+UPDATE reservation
+SET number_of_guests = '3', contact_phonenumber = '56789034' 
+WHERE id = 3;
+
+/*Delete a review with any id, fx 1*/
+
+DELETE FROM reservation WHERE id = 3;
+
+/*Get all reviews*/
+SELECT * FROM review;
+/*Add a new review*/
+insert into review(`title`,`description`,`meal_id`,`stars`)
+values('yummy','yummy spicy chicken curry','1','3');
+/*Get a review with any id, fx 1*/
+SELECT * FROM review WHERE id = 3;
+
+/*Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes*/
+UPDATE review
+SET title = 'chicken curry', stars = '5' 
+WHERE id = 3;
+
+/*Delete a review with any id, fx 1*/
+DELETE FROM review WHERE id = 1;
+
+---Additional queries
+/*Get meals that has a price smaller than a specific price fx 90*/
+SELECT * FROM meal WHERE price > 120;
```

Comment:
Hi @varsha84 ,
This query should be `SELECT * FROM meal WHERE price < 120; instead of SELECT * FROM meal WHERE price > 120;`.
this symbol (<) means smaller than and this one (>) means greater than.

Reply:
it was mistake. I fix it